src/SigningUtil/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using NBitcoin;
+
+namespace SigningUtil
+{
+    internal static class Program
+    {
+        private const string dening14PrivateKey = "KwDiBf89QgGbjEhKnhXJuH7SUWPU37sPj2c2ionJYMnWFj1ZLAiB";
```

Comment:
Do we need this code in production? 
May be it's better to send separate program to Alex instead of storing this code in our source control?

Reply:
Yes we need. I would like to create a new repo in GitHub for storing source code of this things.